docs/src/dMCDA.md

```diff
@@ -1 +1,159 @@
+
+|Parameter                    |Description                                                                        |
+|-----------------------------|-----------------------------------------------------------------------------------|
+|```heat_stress```            |Weight for heat stress when selecting seeding or shading sites.                    |
+|```wave_stress```            |Weight for wave stress when selecting seeding or shading sites.                    |
+|```shade_connectivity```     |Weight for larval connectivity when selecting shading sites.                       | 
+|```in_seed_connectivity```   |Weight for in-degree larval connectivity when selecting seeding sites.             |
+|```out_seed_connectivity```  |Weight for out-degree larval connectivity when selecting seeding sites.            |
+|```coral_cover_high```       |Weight of coral cover area covered when selecting shading sites.                   |
+|```coral_cover_low```        |Weight of coral cover area available for coral gorwth when selecting seeding sites.|
+|```seed_priority```          |Weight of being a priority predecessor site to a priority site when seeding.       |
+|```shade_priority```         |Weight of being a priority predecessor site to a priority site when shading.       |
+|```zone_seed```              |Weight of being in/connected to a site in a zone when seeding.                     |
+|```zone_shade```             |Weight of being in/connected to a site in a zone when seeding.                     |
+|```coral_cover_tol```        |% of area of corals to be seeded allowed on a site when selecting sites for seeding|
+|```deployed_coral_risk_tol```|Risk tolerance for heat and wave risk when seeding or shading corals.              |
+|```depth_min```              |Minimum depth of sites when selecting for seeding or shading corals.               |
+|```depth_offset```           |depth_min+depth_offset is maximum depth of sites when seeding or shading corals.   |
+
+## MCDA methods
+Multi-criteria decision analysis encompasses a series of methods for conducting decision making in a way that is formalised, structured and transparent. It evaluates the decision objective according to numerical measures of decision criteria assembled by the decision maker. Constructing explicit criteria over which to evaluate alternatives allows a transperant evaluation of benefits, negatives and trade-offs in coming to a decision solution.
+
+Commonly, comparison of criteria in MCDA is acheived through the construction of a “values matrix” or “decision matrix”, which contains measures of value for each alternative against an array of criteria. The measure of value, for example, could be a relative ranking of worth, probability of success, or risk of failure. Formally, these values form an $N$ by $M$ matrix where entry $x_{i,j}$ is the value of the $i$th alternative as evaluated against the $j$th criterium. The decision matrix is generally normalised, so that
```

Comment:
> “values matrix” or “decision matrix”

Small nitpick - this sentence has unicode characters. Please replace double quotes with plain text versions.